adapter-base-quarkus/src/main/java/org/eclipse/hono/adapter/quarkus/AbstractProtocolAdapterApplication.java

```diff
@@ -127,8 +125,10 @@
     protected ApplicationConfigProperties appConfig;
 
     @Inject
-    @Named("default-kafka-broker")
-    protected Instance<Map<String, Object>> kafkaConfig;
+    protected Instance<KafkaProducerConfigProperties> kafkaProducerConfig;
```

Comment:
why do we need to use *Instance* here instead of a field of type *KafkaProducerConfigProperties*?

Reply:
We don't. I changed it.